collections/wikitree.js

```diff
@@ -167,7 +167,7 @@ function parseWikiTree(htmlstring, familymembers, relation) {
 
                 for (var i = 0; i < cells.length; i++) {
                     var urlset = $(cells[i]).find('a');
-                    if (exists(urlset)) {
+                    if ((exists(urlset))&&(urlset.length > 0)) {
```

Comment:
Spaces please.

```suggestion
                    if ((exists(urlset)) && (urlset.length > 0)) {
```

Reply:
Also - that's a very "old" way of comparing things in JS;

Today, `if` would evaluate all the following as `false`:
```
- undefined
- null
- "" (empty string) - like in this case
- false
```

So if any, i'd even recommend:
```suggestion
                    if (urlset) {
```

Reply:
Certainly, the suggestion is elegant.. however, in debugging, urlset exists with length = 0. So I'm not sure that it will be effective.
I just tried, it doesn't work.
The problem probably comes from the instantiation of urlset.
We'll have to start all over again.